src/spectrecoff-cli/commands/CanvasImage.fs

```diff
@@ -0,0 +1,69 @@
+﻿namespace SpectreCoff.Cli.Commands
+
+open System.Net.Http
+open Spectre.Console
+open Spectre.Console.Cli
+open SpectreCoff
+open SixLabors.ImageSharp.Processing
+
+type CanvasImageSettings() =
+    inherit CommandSettings()
+
+type CanvasImageExample() =
+    inherit Command<CanvasImageSettings>()
+    interface ICommandLimiter<CanvasImageSettings>
+
+    override _.Execute(_context, _settings) =
+        task {
+        use client = new HttpClient()
+        let! response = client.GetByteArrayAsync("https://sample-videos.com/img/Sample-png-image-500kb.png")
+        let image = canvasImage (Bytes response)
+        let rotatedImage = (canvasImage (Bytes response)).Mutate(fun ctx -> ctx.Rotate(45f) |> ignore)
+        Many [
+           P "Print an image directly to the console!"
+           image |> toOutputPayload
+           CO [P "Take full advantage of the"; LinkWithLabel ("ImageSharp", "https://github.com/SixLabors/ImageSharp"); P "to manipulate your images!"]
+           P "For example, rotate the image by 45°:"
+           rotatedImage |> toOutputPayload
+           ] |> toConsole
+        }
+        |> Async.AwaitTask
+        |> Async.RunSynchronously
+        0
+
+type CanvasImageDocumentation() =
+    inherit Command<CanvasImageSettings>()
+    interface ICommandLimiter<CanvasImageSettings>
+
+    override _.Execute(_context, _settings) =
+        Cli.Theme.setDocumentationStyle
+        NewLine |> toConsole
+        pumped "CanvasImage module"
+        |> alignedRule Left
+        |> toConsole
+
+        Many [
+            CO [
+                C "This submodule provides functionality from the canvas image widget of Spectre.Console ("
+                Link "https://spectreconsole.net/widgets/canvas-image"
+                C ")"
+            ]
+            NL
+            C "The canvas image can be used using the canvasImage function:"
+            BI [
+                P "canvasImage: ImageSource -> CanvasImage"
```

Comment:
The documentation of the extension method to transform it into a payload is missing.

Reply:
Done